<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyCategory</class>
 <widget class="QWidget" name="ModifyCategory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModifyCategory</string>
  </property>
  <widget class="QPushButton" name="SubmittDeleteCAtegor">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submitt</string>
   </property>
  </widget>
  <widget class="QComboBox" name="OfferDiscountName">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>320</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Discount&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="AddNewCategoryName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Delete Category&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submittOfferDiscount">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submitt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Add New Category&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>320</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DeleteCategoryName">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="OfferDiscountBox">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>360</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>270</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Offer Discount&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitAddNewCategory">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submitt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Backbuttom">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>560</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Status Bar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="StatusBar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>560</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Backbuttom</sender>
   <signal>clicked()</signal>
   <receiver>ModifyCategory</receiver>
   <slot>Backbutton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>755</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submitAddNewCategory</sender>
   <signal>clicked()</signal>
   <receiver>ModifyCategory</receiver>
   <slot>AddCategory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SubmittDeleteCAtegor</sender>
   <signal>clicked()</signal>
   <receiver>ModifyCategory</receiver>
   <slot>DeleteCategory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submittOfferDiscount</sender>
   <signal>clicked()</signal>
   <receiver>ModifyCategory</receiver>
   <slot>OfferCategoryDiscount()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Backbutton()</slot>
  <slot>AddCategory()</slot>
  <slot>DeleteCategory()</slot>
  <slot>OfferCategoryDiscount()</slot>
 </slots>
</ui>
